Design: hand-in-page added

RaiteiRyuu opened the pull request:
design of hand in page added to the adobe XD

RaiteiRyuu committed:
Design: hand-in-page added

design of hand in page added to the adobe XD

Abooow commented:
- There is no button to continue. You can add a button at the bottom of the page.
- I think it might be better if the tracking number text is white and centered 
![image](https://user-images.githubusercontent.com/54490104/103776162-58e4b280-502f-11eb-978a-db8e404b26cd.png)
___
Overall it looks good, you have everything that is needed for this page.
Everything is kind of squished together, it could need some spacing. But that can be changed in the future.

timothytin commented:
Overall it is good. If something needs to be changed it can be done later

RaiteiRyuu commented:
Changes:
White text instead of Black in tracknumber
More spacing added to the items inside the frame
Added a button at the bottom.  (usually no button)

RaiteiRyuu committed:
added some changes to design

White text instead of black tracking number
button added at bottom
more spaceing in the main frame
